### PR TITLE
Want to merge user_interface to development

### PR DESCRIPTION
**Pull request by Kingobaino1**
- Get player names
- Display player names
- Choose a symbol
- Player choice of placement of symbol in the board
- Draw the game board
- Display congratulatory message

### CODE REVIEW

File: bin/main [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+puts '+---+---+---+'
+puts '| x | o | o |'
+puts '+---+---+---+'
+puts '| o | x | o |'
+puts '+---+---+---+'
+puts '| x | x | o |'
+puts '+---+---+---+'
+
+puts "\nCongratulations #{player_one}, you won the game!"
```
Comment (voscarmv) [inline bin/main]: Please also add an example display after this, for when there is a draw.

Comment (voscarmv) [inline bin/main]: Please wrap this sequence inside a `while` loop which takes a `game_on` flag variable to end:

```
game_on = true
while game_on
...
# Terminate the game where appropriate by making game_on = false
end

**Review by voscarmv — COMMENTED** [review]
# Impressive work, team! 🖥️ 👀 ✨ 

- Your game interface is almost complete.

## However, some changes are still required... 🧰 🔨 🔧 

- Please read my code comments to find out how to fix your code.

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, team! 🖥️ 👀 ✨ 

- Your game interface is almost complete.

## However, some changes are still required... 🧰 🔨 🔧 

- Please read my code comments to find out how to fix your code.

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Kingsley Ibeh & Guillain Bisimwa** 
Great job following the improvements from the previous review :muscle:  :muscle: 
Your project meets most of the requirements :clap: :clap: :clap:  
We still need to fix some issues:

- [x] game should inform player if the selected move is a winning move
- [x] game should inform the player if the selected move is a draw move
- [x] fix rubocop offenses
- [x] names of the two players should be different

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by krys2fa — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
## Nicely done @guillainbisimwa and @Kingobaino1  👏 
You made all the changes asked for in the previous review. 👍 

However, you still have some rubocop offenses in your code as shown in the screenshot below:

Screen Shot 2020-10-13 at 4 29 43 PM

Please fix them and request another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Comment by guillainbisimwa** [comment]
Please note that `#!/usr/bin/env ruby` cannot be unfrozen and must be at the very top. Therefore the rubocop cannot pass in the local machine.

**Review by SafaErden — APPROVED** [review]
## Status: Approved 🥳🎉🎊

**Hi TEAM, Nice job implementing the required changes! The project looks amazing. I know that takes a lot of time and patience.**
##

**Take into account the following before merging your PR🧐:**

I still get the linter error on local, but since you have the lastest version of linter files and your project passes the online linter checks, I am approving this project.

image


##

Questions or comments? Reach me on [Twitter](https://twitter.com/safaerden) or [LinkedIn](https://www.linkedin.com/in/safaerden/) 🗨️

Happy coding_ 👨‍💻 

**Safa ERDEN**

**Pull request merged**